src/bun.js/webcore/body.zig

```diff
@@ -371,11 +372,22 @@ pub const Body = struct {
             }
         }
 
+        fn _lockedSize(this: *const Value) Blob.SizeType {
+            if (this.Locked.readable) |readable| {
+                return readable.ptr.Bytes.size_hint;
+            }
+
+            return this.Locked.size_hint;
+        }
+
         pub fn size(this: *const Value) Blob.SizeType {
             return switch (this.*) {
                 .Blob => this.Blob.size,
                 .InternalBlob => @as(Blob.SizeType, @truncate(this.InternalBlob.sliceConst().len)),
                 .WTFStringImpl => @as(Blob.SizeType, @truncate(this.WTFStringImpl.utf8ByteLength())),
+                .Locked => {
```

Comment:
```suggestion
                .Locked => this._lcokedSize()
```

Reply:
done in https://github.com/oven-sh/bun/pull/4127/commits/4676cfe8c877ca99e2fb4f2598afb51b51ceaadc